Automattic/wp-calypso
# Pricing page rework: Decouple card components from the product logic

#### Proposed Changes

In continuation of #67485, this PR improves the card components to make them unaware of the product-related logic. It will allow us to extract these components and make them re-usable elsewhere.

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing?flags=jetpack/pricing-page-rework-v1`
    * or boot up this PR 
        * Run `git fetch && git checkout update/jp-pricing-rework/decouple-card-components`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1)
- Confirm that everything is same as before

#### Pre-merge Checklist






Related to #
